include/mbedtls/ssl.h

```diff
@@ -2536,8 +2536,9 @@ int mbedtls_ssl_read( mbedtls_ssl_context *ssl, unsigned char *buf, size_t len )
  *                 a new connection; the current connection must be closed.
  *
  * \note           When this function returns MBEDTLS_ERR_SSL_WANT_WRITE/READ,
- *                 it must be called later with the *same* arguments,
- *                 until it returns a positive value.
+ *                 there may be some partial data in the output buffer. However
+ *                 this partial data is not sent, this function must be called
+ *                 later with the *same* arguments, until it returns a positive value.
```

Comment:
This is talking only about sending, but the top says also `READ`. Please revise.